code/game/mecha/equipment/tools/janitor_tools.dm

```diff
@@ -0,0 +1,356 @@
+// Mecha mop, light replacer, mecha spray, garbage bag
+
+/obj/item/mecha_parts/mecha_equipment/janitor
+
+/obj/item/mecha_parts/mecha_equipment/janitor/can_attach(obj/mecha/janitor/M)
+	if(..() && istype(M))
+		return TRUE
+
+// Mop
+
+#define MOP_SOUND_CD 2 SECONDS //! How many seconds before the mopping sound triggers again
```

Comment:
Move the autodoc above the define please, they're preferred over end-line autodoc comments iirc